src/components/filter/FilterBoilerplate.vue

```diff
@@ -642,21 +635,6 @@ export default {
       margin: 0 0.5rem;
     }
 
-    .filter--dark &__search {
-      color: $light;
-
-      .search-form-control__input,
-      .search-form-control__addon__submit:last-of-type {
-        background: #000;
-        color: inherit;
-      }
-
-      .search-form-control__input:not(:focus),
-      .search-form-control__addon__submit:last-of-type {
-        border-color: #000;
-      }
-    }
```

Comment:
@pirhoo to be clear, I should also add this back? won't it overwrite the css in the subcomponent?

Reply:
Nop, this part is now handle by the search-form-control component and triggered by the dark property you pass down.

Reply:
okay, I misunderstood, sorry